### DISCUSSION TITLE
Add.file annot symbol

### DISCUSSION THREAD

**Discussion by Mano2731995** [pymupdf/PyMuPDF#2165] [Looking for help] [answered]
Hai jorj

I need one help

I hv using page. add_file_annot.

In that condition output was yellow color comment symbol💬

Is there any possible to change yellow color to White color or other colors? 
Or else Is there any possible to hide the symbol? 
My requirement is no need symbol 
FYR I have attached image

Kindly give us solution

![IMG_20230109_162705.jpg](https://user-images.githubusercontent.com/117537492/211293699-fe4981c3-8750-4b54-85c2-fa71572a74c4.jpg)
![IMG_20230109_162615.jpg](https://user-images.githubusercontent.com/117537492/211293950-e1a98765-4f75-47d8-9cbc-97a8c0bc79fb.jpg)
![IMG_20230109_162322.jpg](https://user-images.githubusercontent.com/117537492/211293980-953ead95-fd03-4039-920d-02ccb36bc977.jpg)

**Comment by JorjMcKie** [comment, marked as answer]
Sure, do the following:
```python
flags=a.flags
flags |= fitz.PDF_ANNOT_IS_HIDDEN
a.set_flags(flags)
# no a.update(), respectively after any update()
doc.save("x.pdf")
```

**Reply by Mano2731995** [reply]
I have tried fitz.PDF_ANNOT_IS_HIDDEN
But button(file attachment)also hidden. I need to remove yellow comment symbol only

**Reply by JorjMcKie** [reply]
Aha!
This symbol comes from a PopUp annotation for that annotation. Use `a.delete_responses()` to remove it. There is no other way - it cannot be made invisible, because PDF viewers decide, how they show PopUps if present.
